app/controllers/base_controller.rb

```diff
@@ -17,4 +17,9 @@ def current_archived_intake
       existing || StateFileArchivedIntake.create(email_address: email, contact_preference: "email")
     end
   end
+  def is_intake_locked
+    if current_archived_intake.nil? || current_archived_intake.access_locked? || current_archived_intake.permanently_locked_at.present?
```

Comment:
why is intake considered locked if `current_archived_intake` does not exist?

Reply:
it seems odd for someone to see "This account has been locked due to too many failed login attempts." when that happens ^ Maybe this should lead to a different page

Reply:
Thats a great point. I agree. Although it is not the normal use case to lose your cookies halfway through the flow. Maybe a separate story for this?

Reply:
https://codeforamerica.atlassian.net/browse/FYST-2188